### ISSUE TITLE
Telekom Brand Header: Attribute `aria-current` persists on inactive language switcher items

### ISSUE THREAD

**Issue by georgwittberger-telekom-mms** [telekom/scale#2004]
**Scale Version**

3.0.0-beta.134

**Framework and version**

React with @telekom/scale-components-react

**Current Behavior**

When changing the `active` attribute of `scale-telekom-nav-item` inside the language switcher from `true` to `false` the HTML attribute `aria-current="true"` persists on the inner `a` or `button` element. Thus, the item stays highlighted and screen readers would announce it as the active entry.

![ul9d90aCfq](https://github.com/telekom/scale/assets/64543811/667fae53-93d9-4074-a517-35817e78b3e0)

**Expected Behavior**

When the attribute `active` on `scale-telekom-nav-item` is changed from `true` to `false` the HTML attribute `aria-current="true"` should be removed from the inner `a` or `button` element.

**Code Reproduction**

```html
<scale-telekom-nav-list slot="lang-switcher" variant="lang-switcher" alignment="right" aria-label="Language switcher">
  <scale-telekom-nav-item>
    <button type="button" title="English" aria-label="Switch to English" role="menuitem">EN</a>
  </scale-telekom-nav-item>
  <scale-telekom-nav-item active>
    <button type="button" title="German" aria-label="Switch to German" role="menuitem">DE</a>
  </scale-telekom-nav-item>
</scale-telekom-nav-list>
```

Then remove attribute `active` from `scale-telekom-nav-item` of German language or set it to `false`.

**Comment by felix-ico** [comment]
this bugfix should be available as of beta.139
